romancal/photom/photom_step.py

```diff
@@ -0,0 +1,72 @@
+#! /usr/bin/env python
+
+from romancal.stpipe import RomanStep
+from romancal.photom import photom
+import roman_datamodels as rdm
+
+__all__ = ["PhotomStep"]
+
+
+class PhotomStep(RomanStep):
+    """
+    PhotomStep: Module for loading photometric conversion information from
+        reference files and attaching to the input science data model
+    """
+
+    spec = """
+        source_type = string(default=None)  # Process as specified source type.
+    """
+    reference_file_types = ['photom']
+
+    def process(self, input):
+        """Perform the photometric calibration step
+
+        Parameters
+        ----------
+        input : Roman level 2 image datamodel
+            input roman datamodel
```

Comment:
Should we mention the level 2 schema name (.e.g. wfi_image-1.0.0)?

Reply:
Hrm, I don't have an issue with it, but how to deal with the version number at the end?

Reply:
Maybe wfi_image-1.x.x?

Reply:
Changed.